### PR TITLE
assisted-service: Update from to centos:stream8

### PR DESCRIPTION
**Pull request by lranjbar**
Update the centos image to be `centos:stream8` for assisted-service.

Related to MGMT-6553... probably will still need Dockerfile changes after this.

Signed-off-by: Lisa Rashidi-Ranjbar <lranjbar@redhat.com>

### CODE REVIEW

**Comment by lranjbar** [comment]
/assign @YuviGold 

@YuviGold @osherdp @flaper87 

This is ready for review... it is using the stream8:8 mirror in the ci registry properly now. I think will need to rerun the tests in the PR https://github.com/openshift/assisted-service/pull/1788

**Review by osherdp — CHANGES_REQUESTED** [review]
Same thing for ocm-2.3 branch, no?

**Comment by osherdp** [comment]
/hold
Until figuring out what to do with other major branch

**Comment by osherdp** [comment]
I'm not sure it have any value doing it for ocm-2.3 branch, if it will then it will be pretty easy

**Comment by lranjbar** [comment]
> I'm not sure it has any value doing it for ocm-2.3 branch, if it will then it will be pretty easy

thank you 😃

**Comment by flaper87** [comment]
> OK, let's not get blocked on it

Did you keep the `hold` on purpose?

**Comment by osherdp** [comment]
I never unhold someone else PR
There could always be some other dependencies, checks that needs to be made right after the merge, etc.
@lranjbar, please unhold whenever you want

**Comment by flaper87** [comment]
> I never unhold someone else PR

You held it in the first place, hence my question

> @lranjbar, please unhold whenever you want

I was expecting to see this in your previous comment, hence my question :wink: 

All good, thanks for clarifying :+1:

**Comment by osherdp** [comment]
@flaper87 please see the review approval message, my friend 
Actually, it's a bummer that the hold lock is just one per PR. Maybe having one per user can make those things clearer

**Comment by flaper87** [comment]
> @flaper87 please see the review approval message, my friend

OOOOHHHHHH JEEEEEEEEEEZZZZ!!! I'm really sorry, totally missed it :facepalm: 
Forgive my blindness :pray: 

> Actually, it's a bummer that the hold lock is just one per PR. Maybe having one per user can make those things clearer

Funnily enough, I was thinking the same thing after my last comment.

thanks again for clarifying :pray:

**Comment by lranjbar** [comment]
/unhold

@flaper87 @osherdp Mostly I just like to unhold things in the morning and not before sign off 😅

**Pull request merged**
